packages/i18n/README.md

```diff
@@ -23,18 +23,28 @@ sprintf( _n( '%d hat', '%d hats', 4, 'text-domain' ), 4 );
 // 4 hats
 ```
 
-Note that you will not need to specify [domain](https://codex.wordpress.org/I18n_for_WordPress_Developers#Text_Domains) for the strings.
 
-## Build
+## Build and Usage
 
-You can use the [WordPress i18n babel plugin](/packages/babel-plugin-makepot/README.md) to generate a `.pot` file containing all your localized strings.
+You can use the [wp-cli utility](https://wp-cli.org/) to generate a `.pot` file containing all your localized strings.
 
-The package also includes a `pot-to-php` script used to generate a php files containing the messages listed in a `.pot` file. This is useful to trick WordPress.org translation strings discovery since at the moment, WordPress.org is not capable of parsing strings directly from JavaScript files.
+```sh
+wp i18n make-pot ./src myplugin.pot
+```
+
+Use the [po2json](https://github.com/mikeedwards/po2json) npm module to convert a po file to the proper Jed json format.
```

Comment:
Same reservations here about recommending po2json instead of WP-CLI.

Reply:
Ticket opened here #14093